prime-angular-frontend/src/app/core/services/utils.service.ts

```diff
@@ -133,4 +133,27 @@ export class UtilsService {
   public downloadToken(token: string): void {
     window.location.assign(`${this.config.documentManagerUrl}/documents/downloads/${token}`);
   }
+
+  /**
+   * @description
+   * Opens a mailto href from your current window
+   */
+  public mailTo(recipient: string, subject: string = null, body: string = null) {
+    let href = 'mailto:';
```

Comment:
When there's not recipient the email won't be sent so we could move the assignment of `href` within the if-statement and use interpolation of mailto and recipient.